packages/tracing/test/browser/browsertracing.test.ts

```diff
@@ -379,5 +380,65 @@ describe('BrowserTracing', () => {
         expect(metaTagValue).toBe(content);
       });
     });
+
+    describe('getHeaderContext', () => {
+      it('correctly parses a valid sentry-trace meta header', () => {
+        document.head.innerHTML = `<meta name="sentry-trace" content="12312012123120121231201212312012-1121201211212012-0">`;
+
+        const headerContext = getHeaderContext();
+
+        expect(headerContext).toBeDefined();
+        expect(headerContext!.traceId).toEqual('12312012123120121231201212312012');
+        expect(headerContext!.parentSpanId).toEqual('1121201211212012');
+        expect(headerContext!.parentSampled).toEqual(false);
+      });
+
+      it('returns undefined if the header is malformed', () => {
+        document.head.innerHTML = `<meta name="sentry-trace" content="12312012-112120121-0">`;
```

Comment:
L:

💡 If we see a meta tag that is half-way there, `<meta name="sentry-trace" content="some-invalid-data">` we should log an error / send a usage error to Sentry.

Such invalid tags are most likely coming from a wrong setup in the backend.